src/device-registry/config/constants.js

```diff
@@ -49,6 +49,8 @@ const stageConfig = {
 };
 
 const defaultConfig = {
+  BIG_QUERY_LOCATION: process.env.BIG_QUERY_LOCATION,
```

Comment:
new constants fetching variables from .env

Reply:
Please note that you must update your .env as highlighted in [Slack](https://airqo.slack.com/archives/C03218Z31JT/p1655974243775249). If you cannot reach that Slack message, please reach out to @123MwanjeMike for assistance.